tftb/tests/base.py

```diff
@@ -13,9 +13,29 @@
 from scipy import angle
 from tftb.utils import is_linear
 
+# yoder:
+# let's add at least some backwards python2.x compatibility for now.
+import sys
+#py_ver = sys.version_info.major
+# and assume only backwards revisions (for now):
+ispy2 = (sys.version_info.major<3)		# maybe we should work this into TestBase?
 
 class TestBase(unittest.TestCase):
 
+    # yoder: add __init__()
+    def __init__(self, *args, **kwargs):
+    	# handle various bits, including some python2-3 compatibility, then execute base __init__ as super()
+    	if not ispy2:
+    		# re-map some function calls:
+    		self.assertItemsEqual = self.assertCountEqual		#(inherited from unittest.TestCase)
+    		# ... and others...
+    		#
+    	# and let's go backwards as well, just to be sure (now, we can correct all the downstream code and remove this hack at a later time...).
+    	if ispy2:
+    		self.assertCountEqual = self.assertItemsEqual
```

Comment:
It's not necessary to rewrite the `__init__` method to add aliases to other methods. This should be in the `setUpClass` method of `TestBase`.